src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/AbstractLocatableCollection.java

```diff
@@ -31,7 +31,7 @@
     /**
      * @param metadata records are sorted using the contained {@link SAMSequenceDictionary}
      */
-    AbstractLocatableCollection(final METADATA metadata,
+    protected AbstractLocatableCollection(final METADATA metadata,
```

Comment:
These are package-private so that subclasses must be placed in the collections package.  Easier to keep everything in one place.